xml/mysql_banners.xml

```diff
@@ -132,6 +138,8 @@
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="2" name="os.version"/>
+    <param pos="0" name="os.certainty" value="0.80"/>
```

Comment:
I felt that some fingerprints can be more certain of OS than others - should I just remove all certainty and let the protocol helper handle it?